test/sol6/kyberIntegration.js

```diff
@@ -296,677 +292,811 @@ contract('Proxy + Network + MatchingEngine + FeeHandler + Staking + DAO integrat
 
-        let link = web3.utils.fromAscii("https://kyberswap.com");
+        await networkProxy.setHintHandler(matchingEngine.address, {from: admin});
 
-        // create network fee campaign
-        await updateCurrentBlockAndTimestamp();
-        let newFee1 = curNetworkFee.add(new BN(1));
-        let newFee2 = curNetworkFee.add(new BN(2));
-        let newFee3 = curNetworkFee.add(new BN(3));
-        await submitNewCampaign(daoContract,
-            1, currentBlock + 2, currentBlock + 2 + minCampPeriod,
-            0, 0, 0, [newFee1, newFee2, newFee3], link, {from: campCreator}
-        );
+        await network.setContracts(feeHandler.address, matchingEngine.address, zeroAddress, {from: admin});
 
-        // create brr camp
         await updateCurrentBlockAndTimestamp();
-        let newBrrData1 = getDataFromRebateAndReward(5000, 5000);
-        let newBrrData2 = getDataFromRebateAndReward(5000, 4900);
-        let newBrrData3 = getDataFromRebateAndReward(6000, 3000);
-        await submitNewCampaign(daoContract,
-            2, currentBlock + 2, currentBlock + 2 + minCampPeriod,
-            0, 0, 0, [newBrrData1, newBrrData2, newBrrData3], link, {from: campCreator}
-        );
-        await Helper.mineNewBlockAt(blockToTimestamp(currentBlock + 2));
-
-        // vote for network fee camp, id: 17
-        // mike & victor voted for option 1, so it is the winning option
-        await daoContract.vote(17, 1, {from: mike});
-        await daoContract.vote(17, 1, {from: victor});
-
-        // vote for brr camp, id: 18
-        // mike & victor voted for option 1, so it is the winning option
-        await daoContract.vote(18, 1, {from: mike});
-        await daoContract.vote(18, 1, {from: victor});
-
-        await Helper.mineNewBlockAfter(blocksToSeconds(minCampPeriod));
+    };
 
-        // delay until epoch 18
-        await updateCurrentBlockAndTimestamp();
-        await Helper.mineNewBlockAt(blocksToSeconds(17 * epochPeriod) + daoStartTime);
-        Helper.assertEqual(18, await daoContract.getCurrentEpochNumber());
-
-        // make a first trade and check data changes as expected
-        // no trade at epoch 17, so can not update new data for DAO from previous network fee + brr camps
-        await tradeAndCheckDataChangesAsExpected(
-            18, // epoch
-            curNetworkFee, // new network fee
-            curBrrData.rewardBps, // new reward
-            curBrrData.rebateBps, // new rebate
-            "gas cost no network fee + brr camp, fallback previous values: " // log message
-        );
+    var testSuite = {
+        "test intergration" : testIntergraitonSetup,
+        "upgrage ability - change KyberProxy": testChangeKyberProxySetup,
+        "upgrade ability - change KyberDao/kyberStaking/feeHandler": testChangeKyberDao,
+        "upgrade ability - change network/KyberStorage": testChangeKyberStorageAndNetwork,
```

Comment:
all titles could be redeploy instead of change.

also please have a separate run for network and separate for storage

Reply:
done